flink-connector-kudu/src/main/java/es/accenture/flink/Utils/Utils.java

```diff
@@ -0,0 +1,447 @@
+        }
+        return res;
+    }
+
+    /**
+     * Deelte all rows of the table until empty
+     *
+     * @param tableName  table name to empty
+     * @throws KuduException In case of exception caused by Kudu
+     */
+    public void clearTable (String tableName) throws KuduException {
+        KuduTable table = client.openTable(tableName);
+        List<RowSerializable> rowsList = readTable(tableName);
+
+        String primaryKey = table.getSchema().getPrimaryKeyColumns().get(0).getName();
+        List<Delete> deletes = new ArrayList<>();
+        for(RowSerializable row : rowsList){
+            Delete d = table.newDelete();
+            switch(getRowsPositionType(0, row).getName()){
+                case "string":
+                    d.getRow().addString(primaryKey, (String) row.productElement(0));
+                    break;
+
+                case "int32":
+                    d.getRow().addInt(primaryKey, (Integer) row.productElement(0));
+                    break;
+
+                case "bool":
+                    d.getRow().addBoolean(primaryKey, (Boolean) row.productElement(0));
+                    break;
+
+                default:
+                    break;
+            }
+            deletes.add(d);
+        }
+        for(Delete d : deletes){
+            session.apply(d);
+        }
+        logger.info("SUCCESS: The table has been emptied successfully");
+    }
+
+    /**
+     * Return a list of columns names in a table
+     *
+     * @param table table instance
+     * @return      List of column names in the table indicated in the parameter
+     */
+    public String [] getNamesOfColumns (KuduTable table){
+        List<ColumnSchema> columns = table.getSchema().getColumns();
+        List<String> columnsNames = new ArrayList<>(); //  List of column names
+        for (ColumnSchema schema : columns) {
+            columnsNames.add(schema.getName());
+        }
+        String [] array = new String[columnsNames.size()];
+        array = columnsNames.toArray(array);
+        return array;
+    }
+
+    public boolean checkNamesOfColumns(String [] tableNames, String [] providedNames) throws KuduTableException{
+        boolean res = false;
+        if(tableNames.length != providedNames.length){
+            res = false;
+        } else{
+            for (int i = 0; i < tableNames.length; i++) {
+                res = tableNames[i].equals(providedNames[i]) ? true : false;
+            }
+        }
+        if(!res){
+            throw new KuduTableException("ERROR: The table column names and the provided column names don't match");
+        }
+        return res;
+    }
+
+    public void insert (KuduTable table, RowSerializable row, String [] fieldsNames) throws KuduException, NullPointerException {
+
+        Insert insert;
+        try{
+            insert = table.newInsert();
+        } catch (NullPointerException e){
+            throw new NullPointerException("Error encountered at opening/creating table");
```

Comment:
I don't think this re-throw adds any value.
If this is a protection against table being null, I would do an explicit null check.